daemon/algod/api/client/restClient.go

```diff
@@ -301,14 +369,6 @@ func (client RestClient) ReadyCheck() error {
 	return client.get(nil, "/ready", nil)
 }
 
-// StatusAfterBlock waits for a block to occur then returns the StatusResponse after that block
-// blocks on the node end
-// Not supported
```

Comment:
I was thinking perhaps a note to tell people what to use, but it is easy enough to read the comments on the new/updated functions so I'm good.